src/components/task/existing-task/task-card/index.jsx

```diff
@@ -48,8 +48,9 @@ function TaskCard({
     dispatch(markAsDone(taskId));
     showSuccessToast(SUCCESS_MESSAGE_TASK_DONE);
   };
-
-  const taskHeaderClasses = classnames({ "text-line-through": isCompleted });
+  const taskHeaderClasses = classnames(styles.header, "fw-700", "text-grey1", {
```

Comment:
Is this the best way format to define a css ?
```suggestion
  const taskHeaderClasses = classnames(styles.header, "fw-700", "text-grey-1", {
```